### PR TITLE
Temporary skip Encointer `check-migrations` (TODO: https://github.com/polkadot-fellows/runtimes/issues/186)

### PR DESCRIPTION
**Pull request by bkontur**
- [X] Does not require a CHANGELOG entry

### CODE REVIEW

File: .github/workflows/check-migrations.yml [hunk 1]
```diff
@@ -106,8 +106,17 @@ jobs:
 
           echo "Flags: $EXTRA_FLAGS"
 
+          if [[ "${{ matrix.runtime.name }}" == "encointer-kusama" ]]; then
```
Comment (ggwpez) [inline .github/workflows/check-migrations.yml:109]: We can just un-require it for merge.

**Comment by ggwpez** [comment]
@bkchr could you please un-require the check-migrations encointer for merge?

**Comment by bkontur** [comment]
> @bkchr could you please un-require the check-migrations encointer for merge?

Haha :D, that's actually what I suggested [before](https://github.com/polkadot-fellows/runtimes/pull/159#issuecomment-1966836443). However, after Basti's [comment](https://github.com/polkadot-fellows/runtimes/pull/178#issuecomment-1971121588), I got the impression that it needs to be done here and not in the GitHub configuration.

Anyway, looks like this hack works: https://github.com/polkadot-fellows/runtimes/actions/runs/8097358985/job/22128262518?pr=207 :D

**Comment by ggwpez** [comment]
Hm i see. I think having it red to still remind of of fixing is still better than having it display green while not working.

**Comment by bkchr** [comment]
> Hm i see. I think having it red to still remind us of fixing is still better than having it display green while not working.

Yeah good idea. Removed the check from the required checks.

**Pull request closed without merging**